src/app/components/Redirect404/Redirect404.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import DocumentTitle from 'react-document-title';
+import PropTypes from 'prop-types';
+import appConfig from '../../data/appConfig';
+
+const Redirect404 = (props, context) => {
+
+  const originalUrl = context &&
+    context.router &&
+    context.router.location &&
+    context.router.location.query &&
+    context.router.location.query.originalUrl;
+
+  const {
+    circulatingCatalog,
+    classicCatalog,
+  } = appConfig;
+
+  return (
+    <div className="redirect404">
+      <div className="redirect404text">
+        <h1>{"We're Sorry..."}</h1>
+        <br />
+        <p>
+          {"You've followed an out-of-date link to our research catalog."}
+          <br />
+          <span className="originalUrlText">
```

Comment:
Maybe make this a `div` since it's supposed to be full width

Reply:
changed